LICENSE

```diff
@@ -229,36 +229,22 @@ MIT licenses
 The following components are provided under the MIT License. See project link for details.
 The text of each license is also included at licenses/LICENSE-[project].txt.
 
-    (MIT License) jquery v2.1.4 (https://jquery.org/license/)
-    (MIT License) dagre-d3 v0.6.1 (https://github.com/cpettitt/dagre-d3)
+    (MIT License) jquery v3.4.1 (https://jquery.org/license/)
+    (MIT License) dagre-d3 v0.8.5 (https://github.com/cpettitt/dagre-d3)
```

Comment:
Yarn.lock has this as 0.6.4

Reply:
good catch, i was looking at current version of dagre rather than dagre-d3